lib/core/src/Cardano/Api/Gen.hs

```diff
@@ -506,9 +504,9 @@ genScriptData =
 
 genExecutionUnits :: Gen ExecutionUnits
 genExecutionUnits = do
-    (Large steps) <- arbitrary
-    (Large mem) <- arbitrary
-    pure $ ExecutionUnits steps mem
+    (Large (steps :: Int)) <- arbitrary
+    (Large (mem :: Int)) <- arbitrary
```

Comment:
Using `Word64` might be the safer choice to preserve the previous behaviour. (`ExecuionUnits` had Word64 fields on 1.30.1)

Reply:
Yes - fixed.